decentraland/helpers/storage.js

```diff
@@ -10,6 +12,15 @@ const EPHEMERAL_ACCOUNT_CREATION_ACTIONS = "EPHEMERAL_ACCOUNT_CREATION_ACTIONS";
 const IS_FIRST_APP_USE = "IS_FIRST_APP_USE";
 const USER_ACTIONS = "USER_ACTIONS";
 
+export const clearAllStorage = async () => {
+  await _clearData(IS_FIRST_APP_USE);
+  await _clearData(EPHEMERAL_ACCOUNT_PRIV_KEY);
```

Comment:
I forget, are we just persisting the private key and not the address? And then we recover the address from the private key?

Reply:
Yes, we are persisiting the priv key:
```javascript
export const retrieveAccount = async () => {
  let account = null;

  const privateKey = await _retrieveData(EPHEMERAL_ACCOUNT_PRIV_KEY);
  if (privateKey != null) account = createFromPrivateKey(privateKey);

  return account;
};
```

Reply:
Cool, that's what I thought. Thanks. Resolving this.